Sources_v3/Controllers/ChannelController.swift

```diff
@@ -381,6 +381,66 @@ public extension ChannelControllerGeneric {
         }
     }
     
+    /// Loads new messages from backend.
+    /// - Parameters:
+    ///   - messageId: ID of the last fetched message. You will get messages `older` than the provided ID.
+    ///   - limit: Limit for page size.
+    ///   - completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
+    ///                 If request fails, the completion will be called with an error.
+    func loadNextMessages(
+        after messageId: MessageId? = nil,
+        limit: Int = 25,
+        completion: ((Error?) -> Void)? = nil
+    ) {
+        guard isChannelAlreadyCreated else {
+            channelModificationFailed(completion)
+            return
+        }
+        
+        guard let messageId = messageId ?? messages.last?.id else {
+            log.error(ClientError.ChannelEmptyMessages().localizedDescription)
+            callback { completion?(ClientError.ChannelEmptyMessages()) }
+            return
+        }
```

Comment:
Hmm, this is a question. Maybe we should just retry the original query and only log the error? Something like:
```
log.error("Failed to load next messages because the channel has no messages locally. Trying to reload all messages from the server.")
``` 
The intention of the user is they want more messages, we should try to get them to her/him. WDYT?

Reply:
So you are talking about the case when there is actually some messages on backend but fetch failed previously?
I'm not sure cause in this case I believe it's not the job of this function and this situation will be handled during `startUpdating` call that will throw some error.

Reply:
OK, let's leave it as is for now and we'll see.